src/Runner.php

```diff
@@ -14,22 +14,50 @@ class Runner
 {
     use IO;
 
-    const VERSION = '0.5.4';
+    const VERSION = '0.6.0';
     const ROBOCLASS = 'RoboFile';
     const ROBOFILE = 'RoboFile.php';
-
+    
+    /**
+     * @var CurrentDir
```

Comment:
I just added doc blocks didn't check it this variables are used. You are right this one and $printer aren't used at all. I will remove them and update the PR